taf/updater/updater_pipeline.py

```diff
@@ -401,29 +403,25 @@ def clone_remote_and_run_tuf_updater(self):
                     )
                 else:
                     self.state.auth_repo_name = auth_repo_name
-
             self.state.users_auth_repo = AuthenticationRepository(
-                self.library_dir,
```

Comment:
When we instantiate the `AuthenticationRepository`, we can specify `library_dir` and `name`, in which case the path is set by joining those two parameters. If we specify the path, we then determine `library_dir` and `name` based on it. This is fine, we can specify all 3, just noting that it should not be necessary